pkgs/applications/kde/kdegraphics-thumbnailers.nix

```diff
@@ -9,6 +9,7 @@ mkDerivation {
     license = [ lib.licenses.lgpl21 ];
     maintainers = [ lib.maintainers.ttuegel ];
   };
+  patches = [ ./kdegraphics-thumbnailers-fix-hang.patch ];
```

Comment:
can we get the details of the bug fix linked here as a reference?

Reply:
Added.